aepsych/server/server.py

```diff
@@ -778,19 +778,19 @@ def tell(self, outcome, config, model_data=True):
             )
 
             for param_name, param_value in config.items():
-                if type(param_value) not in [float, int, bool]:
+                if type(param_value) not in [float, int, bool, str]:
```

Comment:
I think what you're trying to do here is check if param_value is iterable, right? So in that case you should do
```
from collections.abc import Iterable

if isinstance(param_value, Iterable):
...
```

Reply:
Yes, you're right! Than you for the suggestion 🙌🏽